Makefile

```diff
@@ -84,7 +84,7 @@ test:
 	export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${LOCAL_DIR}/lib" && \
 	export LOG_LEVEL=DEBUG && \
 	export RUST_BACKTRACE=1 && \
-	cargo test --features "${ENABLE_FEATURES}" ${EXTRA_CARGO_ARGS} -- --nocapture && \
+	cargo test --features "${ENABLE_FEATURES}" --all --exclude tikv_fuzz ${EXTRA_CARGO_ARGS} -- --nocapture && \
```

Comment:
Why exclude fuzz?

Reply:
It causes CI failures previousy: https://circleci.com/gh/tikv/tikv/14134

Reply:
em, we need to fix this fuzz problem. PTAL @overvenus